test/built-ins/Proxy/construct/call-parametes-new-target.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2017 Aleksey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+es6id: 9.5.14
+esid: sec-proxy-object-internal-methods-and-internal-slots-construct-argumentslist-newtarget
+description: >
+    trap is called with handler object as its context, and parameters are:
+    target, an array list with the called arguments and the NewTarget
+info: >
+    [[Construct]] (argumentsList, newTarget)
+
+    9. Let newObj be Call(trap, handler, «target, argArray, newTarget»).
+---*/
+
+let Target = function() {};
+let NewTarget = function() {};
+
+let handler = {
+    construct: function(_Target, args, _NewTarget) {
+        assert.sameValue(this, handler, "trap context is the handler object");
+        assert.sameValue(_Target, Target, "first parameter is the target object");
+        assert.sameValue(args.length, 2, "arguments list contains all construct arguments");
+
+        let [a, b] = args;
```

Comment:
perhaps we should include it somewhere in `CONTRIBUTING.md`? some new tests are written in es6 and one may think it is because the files are new, not because they test newer features.